app/views/foods/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Search Results </h1>
+
+<h4>Total Hits: <%=@foods.first.hits%></h4>
+<h4>Results on this page: <%=@foods.count%></h4>
+
+<h2>Food Attributes</h2>
+<% n = 0 %>
+<% @foods.each do |food| %>
+    <section id="foods_<%= n%>">
+      <p>Description: <%=food.description%></p>
+      <p>Brand Owner: <%=food.brand_owner%></p>
+      <p>GTIN/UPC code: <%=food.gtin_upc%></p>
+      <p>Ingredients: <%=food.ingredients%></p>
+      <p>_____</p>
```

Comment:
Nice styling touch! a `<hr>` tag will do this as well, but the line will go the whole length of the page.